### ISSUE TITLE
Introduce a 'Sort by' setting for saved articles separate to the existing 'Sort by' setting for feeds

### ISSUE THREAD

**Issue by 9aaron** [spacecowboy/Feeder#451]
### Checklist

- [X] I have used the search function for [**OPEN**](https://github.com/spacecowboy/feeder/issues) issues to see if someone else has already submitted the same feature request.
- [X] I have **also** used the search function for [**CLOSED**](https://github.com/spacecowboy/feeder/issues?q=is%3Aissue+is%3Aclosed) issues to see if the feature was already implemented and is just waiting to be released, or if the feature was rejected.
- [X] I will describe the problem with as much detail as possible.
- [X] This request contains only one single feature, **not** a list of multiple (related) features.

### App version

2.7.1

### Where did you get the app from

Google Play

### Feature or improvement you want

Introduce a 'Sort by' setting for saved articles separate to the existing 'Sort by' setting for feeds.

A user would then be able to for example sort saved articles by 'Oldest first' and feeds by 'Newest first'.

### Why should this be added?

This would better cater to the sorting preferences of someone like me which are: 

- For **feeds**, to sort by '**Newest first**'. This is so that when new unread articles which I would like to review for saving are loaded, there is no need to scroll.
- For **saved articles**, to sort by '**Oldest first**'. This is because I would like to actually read articles in chronological order, so I would prefer to not have to scroll to get to the oldest saved article.

**Comment by 9aaron** [comment]
Saw some discussion in #178 but that had multiple, more complex requests.

**Comment by 9aaron** [comment]
Come to think of it, it would be even better to have the sort button at the top of each feed and the saved article list, and for the sort order to be saved per feed/list.

**Comment by spacecowboy** [comment]
I do not want to save sort order per feed. That will be confusing.

But I'm keeping this open because saved articles could use something custom. Just not sure what.

**Comment by ghost** [comment]
Yes, that would be really great -- not per feed for me either, because my preference does not change per feed, but it would be great to be able to rank new feeds oldest-to-newest, because I check several times a day and that helps to see things as they develop, and in the meantime keep the saved articles newest-to-oldest in order to focus on recent items and not get stuck with super old articles I haven't gotten around to reading in weeks/months.
